### PR TITLE
silence staging notifications

### PR DESCRIPTION
**Pull request by escapewindow**
Just so we don't add noise. We can adjust these if we explicitly want to test notifications in staging.

### CODE REVIEW

File: taskcluster/ci/config.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ taskgraph:
             # string: the name of the repo, for humans
             name: "XPI manifest"
             # regex: to match the base name of the repo
-            project-regex: xpi-manifest$
+            project-regex: xpi-manifest(.git)?$
```
Comment (jmaher) [inline taskcluster/ci/config.yml:16]: not sure why this change is in here- the rest make sense to me

Comment (escapewindow) [reply taskcluster/ci/config.yml:16]: the `$` says this is the end of the string (regex).
Sometimes git repo urls are the repo, e.g. `https://github.com/mozilla-extensions/xpi-manifest`, and sometimes they have a `.git` extension, `https://github.com/mozilla-extensions/xpi-manifest.git`. This allows either to work.

**Review by jmaher — APPROVED** [review]
thanks
